CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [1.1.1] -  2023-05-16
+
+### Fixes
+
+- Upgrade version of spring-security-web to fix vul to 6.0.3
+- Upgrade version of spring boot to fix vul to 3.0.6
+- Update DEPENDENCIES file
+- Update Code of Conduct
+
+### Removed
+
+- Removed charts that be on leading repo
```

Comment:
can you pls reword this? i do not get it.

Reply:
Message changed on https://github.com/eclipse-tractusx/vas-country-risk-backend/commit/cc42b73edb0844348067561c2afcd01d4c0cf143